### ISSUE TITLE
Conflicting team IDs

### ISSUE THREAD

**Issue by warrentc3** [tonywagner/mlbserver#36]
A couple of things I've identified with the m3u downloads.

org filtering (includeOrgs=) to TB, includes Chattanooga Lookouts in return
org filtering (includeOrgs=) to KC or CLE, Colorado Rockies is overridden by org minor league affiliate.

team filtering (includeTeams=) returns only specified MLB teams... additionally their affiliates.

**Comment by tonywagner** [comment]
Good catch! Org filtering should be fixed in https://github.com/tonywagner/mlbserver/commit/fbae4b8af5b05dd3025e5a0a033fbc7b4c93cd37

Team filtering is expected behavior

If includeBlackouts id/logo is still an issue, please include an example.
